### PR TITLE
Deploy repo-specific label_sync and trial with test-infra

### PR DESCRIPTION
**Pull request by spiffxp**
See commits for details. Adding descriptions to some labels already in this repo to verify whether the job takes effect

### CODE REVIEW

File: label_sync/cluster/label_sync_cron_job.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
         spec:
           containers:
             - name: label-sync
-              image: gcr.io/k8s-testimages/label_sync:v20180531-20dbd7ab8
+              image: gcr.io/k8s-testimages/label_sync:v20180806-e93188ccb
```
Comment (fejta) [inline label_sync/cluster/label_sync_cron_job.yaml]: Did you already deploy this?

File: boskos/OWNERS [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
 - krzyzacy
 - sebastienvas
+labels:
+- area/boskos
```
Comment (fejta) [inline boskos/OWNERS]: IMO the OWNERS changes should be in their own PR. Right now the label sync utility only runs once a day.

File: label_sync/cluster/label_sync_cron_job.yaml (hunk 1 above)
Comment (spiffxp) [reply label_sync/cluster/label_sync_cron_job.yaml]: yes

File: boskos/OWNERS (hunk 2 above)
Comment (spiffxp) [reply boskos/OWNERS]: these labels all exist already, so it's not like the utility is adding them, just their descriptions, otherwise yeah I would break this up

and in this case I'd like to run the manual job to confirm this takes before falling back to leaning on the cronjob

**Comment by spiffxp** [comment]
```
spiffxp@spiffxp-macbookpro:label_sync (master %)$ make label_sync-job
gcloud container clusters get-credentials "prow" --project="k8s-prow" --zone="us-central1-f"
Fetching cluster endpoint and auth data.
kubeconfig entry generated for prow.
kubectl apply -f cluster/label_sync_job.yaml
job "label-sync" created
spiffxp@spiffxp-macbookpro:label_sync (master %)$ k logs -l job-name=label-sync
# ...
time="2018-08-07T01:43:14Z" level=info msg="Applying 9 changes" org=kubernetes repo=test-infra
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/prow, area/prow, 0052cc)" client=github
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/greenhouse, area/greenhouse, 0052c)" client=github
time="2018-08-07T01:43:14Z" level=info msg="AddRepoLabel(kubernetes, test-infra, area/label_sync, Issues or PRs related to code in /label_sync, 0052c)" client=github
time="2018-08-07T01:43:14Z" level=info msg="AddRepoLabel(kubernetes, test-infra, area/config, Issues or PRs related to code in /config, 0052cc)" client=github
time="2018-08-07T01:43:14Z" level=info msg="AddRepoLabel(kubernetes, test-infra, area/ghproxy, Issues or PRs related to code in /ghproxy, 0052cc)" client=github
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/planter, area/planter, 0052c)" client=github
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/mungegithub, area/mungegithub, 0052cc)" client=github
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/gubernator, area/gubernator, 0052c)" client=github
time="2018-08-07T01:43:14Z" level=info msg="UpdateRepoLabel(kubernetes, test-infra, area/boskos, area/boskos, 0052cc)" client=github
time="2018-08-07T01:45:01Z" level=fatal msg="failed to update kubernetes" error="failed to list labels: [status code 422 not one of [200], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Label\",\"code\":\"invalid\",\"field\":\"color\"}],\"documentation_url\":\"https://developer.github.com/v3/issues/labels/#update-a-label\"} status code 422 not one of [201], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Label\",\"code\":\"invalid\",\"field\":\"color\"}],\"documentation_url\":\"https://developer.github.com/v3/issues/labels/#create-a-label\"} status code 422 not one of [200], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Label\",\"code\":\"invalid\",\"field\":\"color\"}],\"documentation_url\":\"https://developer.github.com/v3/issues/labels/#update-a-label\"} status code 422 not one of [200], body: {\"message\":\"Validation Failed\",\"errors\":[{\"resource\":\"Label\",\"code\":\"invalid\",\"field\":\"color\"}],\"documentation_url\":\"https://developer.github.com/v3/issues/labels/#update-a-label\"}]"
# ...
spiffxp@spiffxp-macbookpro:label_sync (master %)$ k delete job label-sync
job "label-sync" deleted
```

OK, I'm satisfied the per-repo labels get deployed.  Unfortunately I had typos in some label colors, so not all of the updates took hold.

**Comment by spiffxp** [comment]
https://github.com/kubernetes/test-infra/labels?utf8=%E2%9C%93&q=area+ - descriptions took hold for some of the area/ labels
